services/ui-src/src/components/fields/RadioField.tsx

```diff
@@ -44,6 +44,16 @@ const formatChoices = (
   });
 };
 
+const hintTextColor = (clickAction: string) => {
+  switch (clickAction) {
+    case "qmReportingChange":
+    case "qmDeliveryMethodChange": {
+      return "palette.warn_darkest";
+    }
+  }
+  return "palatte.gray";
+};
```

Comment:
Three nits in one here (ooh, wow, triple!)
1. The curly braces inside the switch statement aren't doing anything here
2. Since `switch` has the `default` syntax for when the input doesn't match any case, and that's exactly what we want, let's use it.
3. I'm pretty sure "palatte.gray" was a typo

```suggestion
const hintTextColor = (clickAction: string) => {
  switch (clickAction) {
    case "qmReportingChange":
    case "qmDeliveryMethodChange":
      return "palette.warn_darkest";
    default:
      return "palette.gray";
  }
};
```

Reply:
1 & 3 fair request. 2. nit